contracts/bridge_test.go

```diff
@@ -123,7 +123,7 @@ func TestBridgeDeployWithKLAY(t *testing.T) {
 
 	chargeAmount := big.NewInt(10000000)
 	bridgeAccount.Value = chargeAmount
-	bridgeAddress, tx, _, err := bridge.DeployBridge(bridgeAccount, backend)
+	bridgeAddress, tx, _, err := bridge.DeployBridge(bridgeAccount, backend, false)
```

Comment:
You will also need to test the true case.

Reply:
This TC is not related with the MintBurnMode. So I think it's ok.
But I also agree with your option which mintBurnMode should be tested.
In klaytn-node-test, I added the TC. And In unit test, I also modified deploy-test code. See `func (bm *BridgeManager) DeployBridgeTest(backend *backends.SimulatedBackend, local bool) `. :)